docs/content/querying/aggregations.md

```diff
@@ -2,17 +2,25 @@
 layout: doc_page
 ---
 # Aggregations
-Aggregations are specifications of processing over metrics available in Druid.
+
+Aggregations can be provided at ingestion time as part of the ingestion spec as a way of summarizing data before it enter Druid. 
+Aggregations can also be specified as part of many queries at query time.
+
 Available aggregations are:
 
 ### Count aggregator
 
-`count` computes the row count that match the filters
+`count` computes the Druid row count that match the filters.
```

Comment:
this was already awkward.   "computes the count of Druid rows that match the filters" feels better.